setup.py

```diff
@@ -63,30 +63,29 @@ def initialize_options(self):
         # Default to /usr/local for Homebrew
         prefix = '/usr/local'
       else:
-        print('Warning: mfg-inspector output is not fully implemented for '
-              'Windows. OpenHTF will be installed without it.')
-        self.skip_proto = True
+        prefix = None
 
-    maybe_protoc = os.path.join(prefix, 'bin', 'protoc')
-    if os.path.isfile(maybe_protoc) and os.access(maybe_protoc, os.X_OK):
+    self.protoc = None
+    if prefix:
+      maybe_protoc = os.path.join(prefix, 'bin', 'protoc')
+      if os.path.isfile(maybe_protoc) and os.access(maybe_protoc, os.X_OK):
         self.protoc = maybe_protoc
-    else:
+      else:
         print('Warning: protoc not found at %s' % maybe_protoc)
         print('setup will attempt to run protoc with no prefix.')
-        self.protoc = 'protoc'
-
-    self.protodir = os.path.join(prefix, 'include')
+    if not self.protoc:
+      self.protoc = find_executable('protoc')
+      pc_path = os.path.dirname(find_executable('protoc'))
+      self.protodir = os.path.abspath(os.path.join(os.path.dirname(pc_path), os.path.pardir, 'lib'))
```

Comment:
I'm not familiar with protos on Windows, but this looks a bit suspicious, in that you're going up two directory levels and assuming that `lib/` is there.

If you expect `lib/` in a certain place on Windows, consider hardcoding that path. This is similar to what we do for Mac/Linux where we check `/usr/local/include/` on Mac and `/usr/include/` on Linux. If needed, you can get the drive path using `os.path.splitdrive(self.protoc)[0]`.

If you actually want to go up two directories then this should be cleaned up as:

```python
self.protoc = find_executable('protoc')
self.protodir = os.path.join(
    os.path.dirname(os.path.dirname(os.path.dirname(self.protoc))), 'lib')
```